common/djangoapps/entitlements/api/v1/views.py

```diff
@@ -310,5 +310,10 @@ def destroy(self, request, uuid):
                 'Entitlement Refund failed for Course Entitlement [%s]. Entitlement is not refundable',
                 str(entitlement.uuid)
             )
+            return Response(
+                status=status.HTTP_400_BAD_REQUEST,
+                data={
+                    'message': 'Entitlement refund failed, Entitlement is not refundable'
```

Comment:
Does this (and other strings in this function) need to be translated?

Reply:
No, we only need to translate user facing strings.  We generally do not translate API messages.